src/langs/en.json

```diff
@@ -46,5 +46,12 @@
   "ENROLL_BUTTON": "Enroll",
   "ENROLL_DESCRIPTION": "Click on the button below to enroll and access this item.",
   "ITEM_LOGIN_HELPER_SIGN_OUT": "You're logged in as {{email}}",
-  "HOME_EMPTY": "You have no elements."
+  "HOME_EMPTY": "You have no elements.",
+  "AUTO_LOGIN_NO_ITEM_LOGIN_ERROR": "This item does not support auto-login",
+  "AUTO_LOGIN_ALREADY_LOGGED_IN": "You are already logged in with another account",
+  "AUTO_LOGIN_SIGN_OUT_AND_BACK_IN": "Sign Out and Log Back In",
+  "AUTO_LOGIN_GO_TO_HOME": "Go to Home",
+  "AUTO_LOGIN_WELCOME_TITLE": "Welcome to this study!",
```

Comment:
Should we be that precised about `study`?

Reply:
No idea, what should we put instead ?